src/styles/calendar.scss

```diff
@@ -0,0 +1,70 @@
+$cell-size: 100px;
+$border-width: 1px;
+$calendar-width: 7 * ($cell-size + $border-width) - $border-width;
+$font-size: 30px;
+
+@mixin day($day) {
+  width: $cell-size;
+  height: $cell-size;
+  border: $border-width solid black;
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: relative;
+  background-color: #eee;
+
+
+  &:hover {
+    cursor: pointer;
+    background-color: #FFBFCB;
+    transform: translateY(-20px);
+    transition: all 0.5s;
```

Comment:
transition should be applied on element itself, not on :hover
and you should specify which properties you want to transform, instead of all